### ISSUE TITLE
termui/linegraph.go: index out of range

### ISSUE THREAD

**Issue by MaxSem** [xxxserxxx/gotop#70]
After running for a few seconds on Windows, gotop crashes:
```
panic: runtime error: index out of range [-47]

goroutine 1 [running]:
github.com/xxxserxxx/gotop/v3/termui.(*LineGraph).Draw(0xc000184750, 0xc0000f38c0)
        .../gotop/termui/linegraph.go:86 +0x1026
github.com/gizak/termui/v3.(*Grid).Draw(0xc0000fca90, 0xc0000f38c0)
        .../go/pkg/mod/github.com/gizak/termui/v3@v3.0.0/grid.go:157 +0x1be
github.com/gizak/termui/v3.Render(0xc00009d100, 0x1, 0x1)
        .../go/pkg/mod/github.com/gizak/termui/v3@v3.0.0/render.go:25 +0x2a0
main.eventLoop(0xc00013a760, 0xd, 0xc000146040, 0x12, 0x94de34, 0xa, 0x7, 0x0, 0x0, 0x0, ...)
        .../gotop/cmd/gotop/main.go:183 +0x271
main.main()
        .../gotop/cmd/gotop/main.go:429 +0x7d1
exit status 2
```

Upon digging in it, I see that it's due to one of CPU load values being over 100%, not sure how to fix yet.

Required information:

- gotop version (`gotop -V`): git master
- The output of `uname -a`: Windows 10
- Terminal emulator (e.g. iTerm or gnome terminal): PowerShell
- Any relevant hardware info: 16 core Ryzen CPU
- tmux version if using tmux: N/A

**Comment by xxxserxxx** [comment]
Thanks for the great report.

Preventing the crash is easy, but what does it mean for a CPU to return more than 100% usage?  I can only think that the `gopsutil` is presenting multiple cores as one value on Windows, rather than representing each core individually.  

Merely masking the values so they aren't over 100 will prevent crashes, but will hide what's really going on in the CPU.  I'll add safety code for now to prevent the crashes, but I'll have to think about how to handle this case so the data presented is accurate.  Maybe it's time to replace `gopsutil` -- it keeps showing up as the source of issues.

**Comment by xxxserxxx** [comment]
This change was released in v3.5.0.  I'm creating a separate ticket to address the question: "what does CPU % usage > 100% mean?"
